contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -2002,3 +2002,21 @@ CREATE OR REPLACE FUNCTION sys.sp_datatype_info_helper(
 RETURNS SETOF RECORD
 AS 'babelfishpg_tsql', 'sp_datatype_info_helper'
 LANGUAGE C IMMUTABLE STRICT;
+
+CREATE OR REPLACE FUNCTION OBJECTPROPERTY(IN object_id INT, IN property sys.varchar)
```

Comment:
These could really be SQL language functions, which are cheaper to execute and offer the possibility of being inlined.

Reply:
Yea thats true. The original plan for this function was to implement the full properties, but was shortened to a stub quickly unblock SSMS functionalities (as that's what was needed). However, we are planning to put a PR up for some properties for OBJECTPROPERTY[EX] soon.